apps/fm/test/unit/fm_test.js

```diff
@@ -102,8 +195,8 @@ suite('FM', function() {
   suite('favorite list', function() {
 
     suiteSetup(function() {
-      favoritesList._save = function() {return true};
-      favoritesList._favList = {};
+      //stubFavoritesList();
```

Comment:
If you don't need this here, and stubFavoritesList is called only once, no need to define that function.